### DISCUSSION TITLE
Welcome to notify-rust Discussions!

### DISCUSSION THREAD

**Discussion by hoodie** [hoodie/notify-rust#141] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by grtcdr** [comment]
Thank you @hoodie for notify-rust, it's helped me create a tool I made and use every single to day, the API provided by your crate is awesome to say the least :heart:

**Reply by hoodie** [reply]
Oh wow, thank you! It makes me super happy to hear that!

I just did this library as my first rust project 7years ago to scratch just such an itch myself. I've met a few people that used notify-rust 
I think I never finished the actual tool itself, but I've heard from a few new rustaceans who use notify-rust in one of their first rust projects every once in a while, and it is very gratifying to see that others find this library useful. 
Have a nice day
